lms/djangoapps/lti_provider/views.py

```diff
@@ -28,7 +31,7 @@
 REQUIRED_PARAMETERS = [
     'roles', 'context_id', 'oauth_version', 'oauth_consumer_key',
     'oauth_signature', 'oauth_signature_method', 'oauth_timestamp',
-    'oauth_nonce', 'user_id'
+    'oauth_nonce', 'user_id', 'resource_link_id', 'lti_version', 'lti_message_type'
```

Comment:
Why so strict rules? Do we really need to mark these params as **required**? I may suggest that some couple of existing LTI courses become unworkable after we deploy this to production.

// cc @dpowell